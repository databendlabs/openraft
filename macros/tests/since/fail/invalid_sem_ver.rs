#[openraft_macros::since(version = "1.0.0..0")]
fn main() {}
