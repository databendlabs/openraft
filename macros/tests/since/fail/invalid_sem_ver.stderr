error: `version`(`1.0.0..0`) is not valid semver.
 --> tests/since/fail/invalid_sem_ver.rs:1:36
  |
1 | #[openraft_macros::since(version = "1.0.0..0")]
  |                                    ^^^^^^^^^^
